packages/cmf/src/middlewares/http/middleware.js

```diff
@@ -86,6 +86,7 @@ export function HTTPError(response) {
 	this.name = `HTTP ${response.status}`;
 	this.message = response.statusText;
 	this.stack = {
+		response,
```

Comment:
response is not a plain object. It's a full of promise.
I would prefer a serialization of it (extract the body and add the info to the stack)  see next comment